src/mut/writers.cpp

```diff
@@ -38,10 +40,11 @@ void writeLine(std::ofstream& myfile,
                float diameter) {
     using std::setw;
 
-    myfile << std::to_string(id) << setw(12) << std::to_string(type) << ' ' << setw(12)
-           << std::to_string(point[0]) << ' ' << setw(12) << std::to_string(point[1]) << ' '
-           << setw(12) << std::to_string(point[2]) << ' ' << setw(12)
-           << std::to_string(diameter / 2.f) << setw(12) << std::to_string(parentId) << '\n';
+    myfile << std::to_string(id) << setw(12) << std::to_string(type) << ' ' << setw(12);
```

Comment:
use `FLOAT_PRECISION_PRINT`, no?

also, do you need a setw per output?

Reply:
On this line there are only `int`s.

> also, do you need a setw per output?

I've tried replacing the 2nd `setw` by a `' '` or by nothing, but it did not end-up being formatted correctly.

Reply:
sigh, I read it wrong, my bad